components/esp_common/src/brownout.c

```diff
@@ -70,6 +81,21 @@ void esp_brownout_init(void)
 
     brownout_hal_config(&cfg);
 
+#ifdef CONFIG_ESP32_BROWNOUT_SHUTDOWN_IF_EARLY
```

Comment:
with the renames in S2 and S3 Kconfig files this should be expanded to check for those as well ie:
```

#if CONFIG_ESP32_BROWNOUT_SHUTDOWN_IF_EARLY || CONFIG_ESP32S2_BROWNOUT_SHUTDOWN_IF_EARLY || CONFIG_ESP32S3_BROWNOUT_SHUTDOWN_IF_EARLY
```
Note that defined() or ifdef is optional here since these are boolean values.
Optionally